setup/default.xml

```diff
@@ -21,6 +21,8 @@
     <entry key='filter.enable'>true</entry>
     <entry key='filter.future'>3600</entry>
 
+    <entry key='engineHours.enable'>true</entry>
```

Comment:
Would it be better to rename it to something like `processing.engineHours.enable`?